public/data/python.json

```diff
@@ -233,5 +233,154 @@
         "author": "dostonnabotov"
       }
     ]
+  },
+  {
+    "categoryName": "JSON Manipulation",
+    "snippets": [
+      {
+        "title": "Read JSON File",
+        "description": "Reads a JSON file and parses its content.",
+        "code": [
+          "import json",
+          "",
+          "def read_json(filepath):",
+          "    with open(filepath, 'r') as file:",
+          "        return json.load(file)",
+          "",
+          "# Usage:",
+          "data = read_json('data.json')",
+          "print(data)"
+        ],
+        "tags": ["python", "json", "file", "read"],
+        "author": "e3nviction"
+      },
+      {
+        "title": "Write JSON File",
+        "description": "Writes a dictionary to a JSON file.",
+        "code": [
+          "import json",
+          "",
+          "def write_json(filepath, data):",
+          "    with open(filepath, 'w') as file:",
+          "        json.dump(data, file, indent=4)",
+          "",
+          "# Usage:",
+          "data = {'name': 'John', 'age': 30}",
+          "write_json('data.json', data)"
+        ],
+        "tags": ["python", "json", "file", "write"],
+        "author": "e3nviction"
+      }
+    ]
+  },
+  {
+    "categoryName": "SQLite Database",
+    "snippets": [
+      {
+        "title": "Create SQLite Database Table",
```

Comment:
I don't know how i feel about this example, it's cool to have snippets about databases, but the fact that the table name is not hardcoded but the fields are gives me a weird vibe

Reply:
I have a similar feeling about the "Insert Data into SQLite Table" snippet

Reply:
I have made it possible to now also change the columns, although I'm not sure if its better this way, or to make it completely hardcoded, since its a little larger now.

Reply:
honestly I like the dynamic one, but yeah it's a bit long... @dostonnabotov could I get your opinion on this ?

Reply:
Dynamic version is definitely preferred. In terms of code length, if the code contains necessary parts and is often written in the same way, it's okay. Since it would mean other developers could easily copy and use them.

Reply:
Seems good to me then